### PR TITLE
multi: Clean up code.

### LINKED ISSUE #5: cw-decred: Clean up.
On the cw-decred branch, remove all unneeded imports. Clean up formatting. Point Unspents to the class in cw_core/unspent_transaction_output.dart and remove the implementation in cw_decred.

### PR DESCRIPTION
**Pull request by JoeGruffins**
closes #5

### CODE REVIEW

File: cw_decred/.gitignore
Comment (itswisdomagain) [inline cw_decred/.gitignore:1]: Why was this file deleted?

File: lib/src/screens/dashboard/widgets/menu_widget.dart [hunk 1]
```diff
@@ -35,6 +35,7 @@ class MenuWidgetState extends State<MenuWidget> {
       this.bitcoinCashIcon = Image.asset('assets/images/bch_icon.png'),
       this.decredIcon = Image.asset('assets/images/decred_menu.png');
 
+
```
Comment (itswisdomagain) [inline lib/src/screens/dashboard/widgets/menu_widget.dart:38]: 2 empty line spaces necessary?

**Review by itswisdomagain — APPROVED** [review]
Looks good. Just some Qs.

File: cw_decred/.gitignore
Comment (JoeGruffins) [reply cw_decred/.gitignore:1]: I wasn't sure nested .gitignore work but I guess they do. Do we need it? Can add back. I don't think it was presently ignoring anything that exists was it?

File: lib/src/screens/dashboard/widgets/menu_widget.dart (hunk 1 above)
Comment (JoeGruffins) [reply lib/src/screens/dashboard/widgets/menu_widget.dart:38]: I think I changed this in the last pr unintentionally. The diff with main should make more sense.

```
$ git diff main -- lib/src/screens/dashboard/widgets/menu_widget.dart
diff --git a/lib/src/screens/dashboard/widgets/menu_widget.dart b/lib/src/screens/dashboard/widgets/menu_widget.dart
index bf001eb3..2e81d4cd 100644
--- a/lib/src/screens/dashboard/widgets/menu_widget.dart
+++ b/lib/src/screens/dashboard/widgets/menu_widget.dart
@@ -32,7 +32,8 @@ class MenuWidgetState extends State<MenuWidget> {
       this.ethereumIcon = Image.asset('assets/images/eth_icon.png'),
       this.nanoIcon = Image.asset('assets/images/nano_icon.png'),
       this.bananoIcon = Image.asset('assets/images/nano_icon.png'),
-      this.bitcoinCashIcon = Image.asset('assets/images/bch_icon.png');
+      this.bitcoinCashIcon = Image.asset('assets/images/bch_icon.png'),
+      this.decredIcon = Image.asset('assets/images/decred_menu.png');
 
 
   final largeScreen = 731;
@@ -54,6 +55,7 @@ class MenuWidgetState extends State<MenuWidget> {
   Image bitcoinCashIcon;
   Image nanoIcon;
   Image bananoIcon;
+  Image decredIcon;
 
   @override
   void initState() {
@@ -219,6 +221,8 @@ class MenuWidgetState extends State<MenuWidget> {
         return nanoIcon;
       case WalletType.banano:
         return bananoIcon;
+      case WalletType.decred:
+        return decredIcon;
       default:
         throw Exception('No icon for ${type.toString()}');
     }
```

File: cw_decred/.gitignore
Comment (itswisdomagain) [reply cw_decred/.gitignore:1]: Don't know. But other cw_* have it, so maybe good to have it?

Comment (JoeGruffins) [reply cw_decred/.gitignore:1]: Yeah it got in because was a copy of cw_bitcoin. I'll add back I guess? Just a blank file for now?

**Comment by JoeGruffins** [comment]
Added an empty .gitignore https://github.com/JoeGruffins/cake_wallet/compare/e7d8195a492beb15934c070f461e40e67707fdd1..18b578dea3a1d63b01b981bc844c04df3518a6f5

**Comment by JoeGruffins** [comment]
Looks like I unintentionally changed a bitcoin file. Reverting it...

**Comment by JoeGruffins** [comment]
Reverted the bitcoin file https://github.com/JoeGruffins/cake_wallet/compare/18b578dea3a1d63b01b981bc844c04df3518a6f5..d83e82dcae55fb9c29610631253f5cc56d0b79a4
